README.md

```diff
@@ -1 +1,20 @@
-# HeaderGraphPartioning
\ No newline at end of file
+# HeaderGraphPartioning
+Scripts used to generate a graph suggesting possible partitions for a header.
+
+Run the following scripts on the first run. If you want to test for different compile_commands.json you will need to rerun this:
+
+```bash
+python process_ast.py -c compile_commands.json > ast_dump.txt
+python ast_json.py -f ./ast_dump.txt
+```
+
+Replace compile_commands.json with the relative or absolute location. 
+Optionally add -c and a filename to the ast_json invocation if you want to specify a file other than the auto generated outfile.json.
```

Comment:
so ast_json.py produces outfile.json that's consumed by hierarchical_agglomeration.py? Can you make that more explicit?

I think it should be possible at some point to merge process_ast.py and ast_json.py and just produce outfile.json (or have hierarchical_agglomeration drive the whole thing.

Reply:
file an issue to track that, the close it via pr